packages/agents/sdk/src/config.ts

```diff
@@ -214,9 +215,10 @@ export const XERC20REGISTRY_DOMAIN_ADDRESS: Record<string, string> = {
   "1650553709": "0xcc1ce162fb1d70a377d1542873b1db57f5ff14d1", // base
   "1836016741": "0x689cCB572262e0ca66146FE5Ed5D998543d3AA3B", // mode
   "1935897199": "0x397aEEEDd44f40326f9eB583a1DFB8A7A673C40B", // scroll
+  "2020368761": "", // xlayer
```

Comment:
Registry + adapter to be deployed in separate PR?

Reply:
yes, Will add it separately in sdk